contrib/helloimage/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker://docker.io/library/alpine:latest
```

Comment:
shouldn't this also use quay.io ? just asking

Reply:
I looked for an "official" alpine on quay.io, but only found "quay.io/someuser/alpine:latest".  I could use the quay.io/libpod/alpine, but I'd rather get one directly from alpine.  Do you know of one @afbjorklund ?

Reply:
I guess quay.io only knows "some" distributions, according to https://github.com/containers/shortnames

So we still have to use Docker, for the regular ones. https://docs.docker.com/docker-hub/official_images/

Unless Quay is planning on some more official mirroring ?

I don't use it, since it didn't seem to have any free plans.

Reply:
Why not use ubi8-micro?

Reply:
I could use ubi8-micro, but it's almost 33MB bigger.  Should I splurge?

Reply:
Splurge, I don't want to be pulling content from docker.io.

Reply:
@fatherlinux WDYT?

Reply:
I guess it would only pull at build time, not a run time. But maybe that's enough ?

Reply:
Depending on your needs, you could even do a static binary (with no shell at all)

Reply:
I think we should keep this simple.  Pulling for 10 more seconds is not a big issue.